src/unum/wireless/mt76xx_common/mt76xx_radios_platform.c

```diff
@@ -21,11 +21,10 @@
 
 // List of platform radios indexed by the radio number
 static char *radio_list[] = {
-    "ra0", // 2.4 GHz radio
-    "rai0" // 5 GHz radio
+    WIFI_RADIO_24, // 2.4 GHz radio
```

Comment:
I don't think we need these defines. You can use:
```
{
WIFI_RADIO_24_PREFIX "0",
WIFI_RADIO_5_PREFIX "0"
}
```

Reply:
Changed.

Reply:
There are no default values as of now.

Reply:
Ok, but you'll need them (the defaults) here in this pull request. All the external builds using the opensource code should still work without the need to change anything in their repos after you merge this pull request. Maintaining backward compatibility is now required for this code.

Reply:
Hi Denis, I think there are no opensource repos based on mt76xx .

Reply:
Added default values.